vignettes/teal.Rmd

```diff
@@ -112,6 +112,11 @@ so your users can view and configure their reports before downloading them.
 
 To learn more about the `teal` framework we recommend first exploring some of the available analysis modules.
 
+## Known bugs
+
+- It could be needed to use `options(shiny.useragg = FALSE)` to make the `teal::init` apps more stable, till `ragg` package is not fixed. It is recommented to put this option in the `.Rprofile` file.
```

Comment:
IMO it would be worth to put an issue reference so that it not some-unknown-bug but actually we know what we are talking about

Reply:
just added https://github.com/rstudio/shiny/issues/3704